docs/integration/alerts-twilio-integration.md

```diff
@@ -0,0 +1,44 @@
+title: Twilio Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Twilio. Use them together to get the information about each notification quickly and on any device.
+
+## In Telegram
+
+**1.** Create a Twilio account at [twilio.com](https://twilio.com).
+
+**2.** Go to your Twilio project that you want to send notifications to and setup a new Twilio **Phone number**. This number will be used as the sender or the SMS messages. If you are testing Twilio you can click the **Get a Trial Number** button to generate a temporary phone number for tests. In addition to the phone number you will also need the **Account SID** and the **Auth Token** to be able to setup the notification in Sematext. Note those down:
```

Comment:
Go to your Twilio project that you want to send notifications to and setup a new Twilio **Phone number**.
==>
Go to the Twilio project where you want to send notifications.  Set up a new Twilio **Phone number**.

or the SMS -- should that be "of the SMS"?

to setup => to set up

Reply:
Adjusted, thanks.